src/bentoml/_internal/frameworks/fastai.py

```diff
@@ -140,7 +140,7 @@ def save_model(
               :obj:`~bentoml.Tag` name.
         learner: :obj:`~fastai.learner.Learner` to be saved.
         signatures: Signatures of predict methods to be used. If not provided, the signatures default to
-                    ``predict``. See :obj:`~bentoml.types.ModelSignature` for more details.
+                    ``predict``. See :obj:`~bentoml._internal.models.model.ModelSignature` for more details.
```

Comment:
Did this not work before?

Reply:
I don't think this shows on doc no?

Reply:
and also it doens't make sense to have a `bentoml.types`, since `bentoml.types` can only work under the `TYPE_CHECKING` anw.

Reply:
Hm, that feels like it shouldn't be the case.

Reply:
Isn't the real problem that the bentoml.types module doesn't have any docs?